### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by daniels-ob** [kifi/franz#76]
There are some issues that need resolving and there's also a pending pull request.  Two of the contributors at kifi haven't made any contributions in the past year, and there has been no activity for kifi/franz sine January 7th, 2017.  

If the owners are no longer interested or too busy to maintain this library, would they be willing to let someone else take ownership?

**Comment by jeffrey-aguilera** [comment]
I had the same question.  Based on the non-response, I would have to say the answer is no.

**Comment by stkem** [comment]
Unfortunately Kifi is no longer in business and you are right, this isn't maintained any more. Anyone interested in taking over?

**Comment by idopreuss** [comment]
Hi, did someone take the ownership?

**Comment by stkem** [comment]
I don't think so. Repeating my question from a years ago: Does anyone want to?

**Comment by idopreuss** [comment]
Hi, if i take ownership, how does the CiCd works? same as now? should i do something special?

**Comment by stkem** [comment]
Kifi doesn't exist any more, so nobody has control over com.kifi. You'll need to publish a new package from a fork, I'm afraid.
